olp-cpp-sdk-dataservice-read/tests/VersionedLayerClientTest.cpp

```diff
@@ -72,4 +72,16 @@ TEST(VersionedLayerClientTest, GetData) {
   }
 }
 
+TEST(VersionedLayerClientTest, RemoveFromCache) {
+  std::shared_ptr<CacheMock> cache_mock = std::make_shared<CacheMock>();
+  olp::client::OlpClientSettings settings;
+  settings.cache = cache_mock;
+
+  VersionedLayerClient client(kHrn, kLayerId, settings);
+  {
+    SCOPED_TRACE("Remove not existing partition from cache");
+    ASSERT_FALSE(client.RemoveFromCache(kPartitionId));
+  }
+}
```

Comment:
Not sure what you are testing here. Where are the expectations? 
Were are the positive tests?